### ISSUE TITLE
Docker install using install script won't start the bc-server container

### ISSUE THREAD

**Issue by brooks-bridges** [bluecherrydvr/bluecherry-docker#29]
Logs are attached.  It appears that there's an issue with rsyslog for some reason.
[bc-server_logs.txt](https://github.com/bluecherrydvr/bluecherry-docker/files/13233181/bc-server_logs.txt)

**Comment by curtishall** [comment]
Can you run this inside the bluecherry-docker directory:

```
sed -i 's/image: bluecherry:latest/image: bluecherry:unstable/g' docker-compose.yml
docker compose pull
docker compose down
docker compose up -d
```

**Comment by curtishall** [comment]
(If you have a license key already installed please deactivate it first)

**Comment by brooks-bridges** [comment]
Appears to be doing the same thing
[bc-server_logs (1).txt](https://github.com/bluecherrydvr/bluecherry-docker/files/13234141/bc-server_logs.1.txt)

**Comment by curtishall** [comment]
I see the problem, a new build should be pushed in the next few minutes.  Try the above commands again shortly.

**Comment by brooks-bridges** [comment]
well, the rsyslog error is gone, but now it appears to be stuck in a loop and stopping at this:

2023-11-04T01:09:23.064816497Z nginx: [emerg] socket() [::]:80 failed (97: Address family not supported by protocol)

Log is attached.
[bc-server_logs (2).txt](https://github.com/bluecherrydvr/bluecherry-docker/files/13255526/bc-server_logs.2.txt)

**Comment by curtishall** [comment]
Did you disable IPv6?  Are you using Linux as the host?

https://forums.docker.com/t/ipv6-disabled-on-my-computer-but-docker-network-seems-looking-for-it/107299

**Comment by brooks-bridges** [comment]
Yes, ubuntu 22, and it is disabled.  Is there any way to change that with nginx?  I really don't want ipv6 enabled on this system as it screws with my openvpn tunnels.

**Comment by curtishall** [comment]
Okay. You can comment out the server line that has [::]... in /etc/nginx/sites-enabled/default 

To get into the container run:

docker compose exec -it bc-server /bin/bash

nano should be installed so once you are in run:

nano /etc/nginx/sites-enabled/default


If that won't work because docker is respawning I'll come up with another solution when I'm back to a computer

**Comment by brooks-bridges** [comment]
Yeah, I can't get into it because as soon as I try to spawn a shell it blows up and dies.  I'm looking through the dockerfile to see if there's a way I can just remove that line from the default file, or just remove the default file entirely.

**Comment by curtishall** [comment]
Can you try to pull the latest and see if that works?

**Comment by brooks-bridges** [comment]
well, after some git repo tomfoolery, I finally got the modified docker-compose.yml and the right build-source repo on my local system, however now it's blowing up with another error and restarting continuously:

_> Update MySQL's my.cnf from environment variables passed in from docker
> Writing /root/.my.cnf
> Update bluecherry server's bluecherry.conf from environment variables passed in from docker
> Writing /etc/bluecherry.conf
> chown bluecherry:bluecherry /var/lib/bluecherry/recordings
chown: cannot access '/var/lib/bluecherry/.local/share/data/': No such file or directory_

just FYI, after changing to the build-source branch in order to check out your changes, the the docker-compose.yml in that branch uses the "citesting" docker image, so I changed that back to unstable.

**Comment by brooks-bridges** [comment]
Ok, so clearing everything out and starting again, by resetting to the master branch and then making manual changes to docker-compose.xml and server/Dockerfile , it appears to be started and running.  I will do some further testing to see what happens with it and ensure it stays running.
